core/network/src/main/kotlin/org/mifospay/core/network/di/NetworkModule.kt

```diff
@@ -177,8 +177,8 @@ class NetworkModule {
 
     @Provides
     @Singleton
-    fun provideAuthenticationService(client: HttpClient): KtorAuthenticationService {
-        return KtorAuthenticationService(client)
+    fun provideAuthenticationService(client: HttpClient, preferencesHelper: PreferencesHelper): KtorAuthenticationService {
```

Comment:
Don't we have the datastore? @AdityaKumdale recently migrated the SharedPrefrence with Datastore.

Reply:
will have to crosscheck this with aditya

Reply:
Yes we have implemented proto datastore instead of preference store

Reply:
I don't see any implementation of datastore in `datastore` module @AdityaKumdale why?